### PR TITLE
Studio: add admon about live.end()

### PR DESCRIPTION
**Pull request by tapadipti**
Add this admon:
image

### CODE REVIEW

File: content/docs/studio/user-guide/projects-and-experiments/live-metrics-and-plots.md [hunk 1]
```diff
@@ -84,6 +84,14 @@ with Live() as live:
   live.end()
 ```
 
+<admon type="warn">
+
+If `live.end()` is omitted, Iterative Studio will continue to wait for new
```
Comment (mvshmakov) [inline content/docs/studio/user-guide/projects-and-experiments/live-metrics-and-plots.md:89]:
```suggestion
If `live.end()` is not called, Iterative Studio will continue to wait for new
```

**Review by mvshmakov — APPROVED** [review]
Nice, thanks!

**Comment by dberenbaum** [comment]
`live.end()` is automatically called by using `with Live()`, so adding it in the example code block and calling it out like this might be redundant/misleading. I would recommend `with Live()` in this scenario over `live.end()` because it will call `live.end()` even if an error is raised.

cc @daavoo

**Comment by daavoo** [comment]
> `live.end()` is automatically called by using `with Live()`, so adding it in the example code block and calling it out like this might be redundant/misleading. I would recommend `with Live()` in this scenario over `live.end()` because it will call `live.end()` even if an error is raised.
> 
> cc @daavoo

Agree. What was the motivation for the callout? 
(and including redundant `live.end`)

If someone encountered unexpected behavior using `with Live()` context manager it might be a bug. The extra `live.end` call should not be needed.

**Comment by tapadipti** [comment]
@dberenbaum @daavoo `with Live()` is working as expected. I'll update the content of the admon and example code block.

**Pull request merged**